src/ODESolvers/AdditiveRungeKuttaMethod.jl

```diff
@@ -327,6 +327,13 @@ ODEs.updatedt!(ark::AdditiveRungeKutta, dt) = ark.dt[1] = dt
 function ODEs.dostep!(Q, ark::AdditiveRungeKutta, param, timeend,
                       adjustfinalstep)
 
+  # this should be removed once we get rid of the old interface
+  if !isnothing(param)
```

Comment:
Can we easily patch the old examples to remove this? (Just wondering)

Reply:
Good idea, thanks. Done.